chemprop/nn/predictors.py

```diff
@@ -147,40 +154,6 @@ class RegressionFFN(_FFNPredictorBase):
     _default_criterion = MSELoss()
     _default_metric = MSEMetric()
 
-    def __init__(
-        self,
-        n_tasks: int = 1,
-        input_dim: int = DEFAULT_HIDDEN_DIM,
-        hidden_dim: int = 300,
-        n_layers: int = 1,
-        dropout: float = 0.0,
-        activation: str = "relu",
-        criterion: LossFunction | None = None,
-        loc: float | Tensor = 0.0,
-        scale: float | Tensor = 1.0,
-    ):
-        super().__init__(n_tasks, input_dim, hidden_dim, n_layers, dropout, activation, criterion)
-
-        if isinstance(loc, float):
-            loc = torch.ones(1, self.n_tasks) * loc
-        else:
-            loc = torch.tensor(loc).view(1, -1)
-        self.register_buffer("loc", loc)
-
-        if isinstance(scale, float):
-            scale = torch.ones(1, self.n_tasks) * scale
-        else:
-            scale = torch.tensor(scale).view(1, -1)
-        self.register_buffer("scale", scale)
-
-    def forward(self, Z: Tensor) -> Tensor:
-        Y = super().forward(Z)
-
-        return self.scale * Y + self.loc
-
-    def train_step(self, Z: Tensor) -> Tensor:
-        return super().forward(Z)
-
 
 @PredictorRegistry.register("regression-mve")
 class MveFFN(RegressionFFN):
```

Comment:
does the `output_transform` work properly here?

Reply:
The uncertainty related modules are scheduled to be addressed in v2.1, so I didn't test if `output_transform` works for the `MveFFN`. I think it might make sense to postpone testing this part to v2.1 as we should also add tests for MVE regression. Currently we don't have integration test or CLI test for MVE regression.